code/obj/item/clothing/helmets.dm

```diff
@@ -722,8 +725,9 @@
 				src.reveal(owner)
 
 	attack_self(mob/user) //let people toggle these inhand too
-		for(var/obj/ability_button/mask_toggle/toggle in ability_buttons)
-			toggle.execute_ability() //This is a weird way of doing it but we'd have to get the ability button to update the icon anyhow
+		if (ishuman(user))
+			for(var/obj/ability_button/mask_toggle/toggle in ability_buttons)
+				toggle.execute_ability() //This is a weird way of doing it but we'd have to get the ability button to update the icon anyhow
```

Comment:
but why do they even have the ability button?

Reply:
x

Reply:
I've hopefully hidden the button.